Task2/select_best_model.py

```diff
@@ -0,0 +1,58 @@
+with open('gridSearchCV.txt', 'r') as grid_searched:
```

Comment:
You should create a module for this, that has function that you can call. Ideally, you should have your predictor pipelined so you can pass the input and test files to the script, and expect the results, without any other intervention.